vulnerabilities-CVEd/CVE-2022-22978-spring_security_auth_bypass.bcheck

```diff
@@ -0,0 +1,18 @@
+metadata:
+    language: v1-beta
+    name: "CVE-2022-22978 Spring Security Authorization Bypass"
+    description: "find CVE-2022-22978 Spring Security Authorization Bypass"
+    author: "timeshatter"
+    tags: "Spring Security Authorization Bypass","CVE-2022-22978"
+
+given request then
+    send request called check :
+        appending path: "%0a"
+
+    if {check.response.status_code} is "200" and not({base.response.status_code} is "200") then
+      report issue:
+        severity: high
+        confidence: firm
```

Comment:
Do you think 'firm' is the correct confidence here? We've not really fingerprinted that the app is using Spring.

Reply:
At present, I don't have a good way to identify spring, so let's set it as tentative